pandas/core/indexes/base.py

```diff
@@ -2320,11 +2318,15 @@ def symmetric_difference(self, other, result_name=None):
         except TypeError:
             pass
 
-        attribs = self._get_attributes_dict()
-        attribs['name'] = result_name
-        if 'freq' in attribs:
-            attribs['freq'] = None
-        return self._shallow_copy_with_infer(the_diff, **attribs)
+        if self.nlevels > 1 and len(the_diff) == 0:
+            return type(self)([[] for _ in range(self.nlevels)],
+                              [[] for _ in range(self.nlevels)])
+        else:
```

Comment:
You can remove this `else` condition, and just dedent everything like it was before.